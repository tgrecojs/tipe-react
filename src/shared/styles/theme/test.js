import React from 'react'
import ThemeComponent from './component';
import renderer from 'react-test-renderer'


test(`<ThemeComponent />`, () => {
  expect(
    renderer.create(<ThemeComponent />).toJSON()
  ).toMatchSnapshot()
})