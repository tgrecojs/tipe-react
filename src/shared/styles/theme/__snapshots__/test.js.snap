// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThemeComponent /> 1`] = `
.emotion-0 {
  width: 96px;
  height: 96px;
  border-radius: 50%;
  border: 1px solid #BF67AD;
}

<img
  className="emotion-0 emotion-1"
  src="https://i.pinimg.com/564x/9b/d7/4f/9bd74fc0e9547d6da161c2c9c6e485e3.jpg"
/>
`;
